_posts/2016-07-29-npm-install-electron.md

```diff
@@ -0,0 +1,67 @@
+---
+title: npm install electron
+author: zeke
+---
+
+With help from the community, we have taken over the `electron` package
+name on npm. As of Electron version 1.3.1, you can
+`npm install electron --save-dev` to install the latest precompiled version
+of Electron in your app.
+
+---
+
+![npm install electron](https://cloud.githubusercontent.com/assets/378023/17259327/3e3196be-55cb-11e6-8156-525e9c45e66e.png)
+
+## The History of Electron Prebuilt
+
+If you've ever worked on an Electron app before, you've likely come across the
+`electron-prebuilt` npm package. This package is an indispensable part of nearly
+every Electron project. When installed, it detects your operating system
+and downloads a prebuilt binary that is compiled to work on your system's
+architecture.
+
+## The Name
+
+The Electron installation process was often a stumbling block for new users.
+Many brave people tried to get started without reading the documentation, and
```

Comment:
This feels kind of like a dig at people for not reading the docs, how about:

> Many brave people tried to get started developing an Electron app by running `npm install electron` instead of `npm install electron-prebuilt`,...